c10/core/TensorOptions.h

```diff
@@ -396,6 +396,9 @@ struct C10_API TensorOptions {
             return DispatchKey::MSNPUTensorId;
           case DeviceType::XLA:
             return DispatchKey::XLATensorId;
+          // IKTODO? Is it right to have (Dense - Vulkan) here ?
```

Comment:
yeah, as mentioned above we probably should map to strided directly. I recall you mentioning earlier that vulkan actually has two different layouts (texture vs buffer) - do we want to expose it to the user or make completely implicit?